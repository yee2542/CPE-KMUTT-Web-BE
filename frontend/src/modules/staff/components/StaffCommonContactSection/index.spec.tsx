import { shallow } from 'enzyme';

import StaffCommonSection from 'modules/staff/components/StaffCommonContactSection';

describe('StaffCommonContactSection Tests', () => {
  it('StaffCommonContactSection  to match snapshot', () => {
    const wrapper = shallow(<StaffCommonSection header="" />);
    expect(wrapper).toMatchSnapshot();
  });
});
