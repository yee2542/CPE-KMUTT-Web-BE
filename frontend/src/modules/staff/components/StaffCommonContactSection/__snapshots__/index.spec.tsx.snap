// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StaffCommonContactSection Tests StaffCommonContactSection  to match snapshot 1`] = `
<Fragment>
  <styled__Container>
    <styled__Header>
      <styled__Title
        variant="h3"
      />
    </styled__Header>
    <styled__Content />
  </styled__Container>
</Fragment>
`;
